src/pynwb/data/nwb.icephys.yaml

```diff
@@ -8,7 +8,7 @@ groups:
   - dims:
     - num_times
     doc: Recorded voltage or current.
-    dtype: number
+    dtype:
```

Comment:
Is this supposed to be an empty dtype?

Reply:
Oops, that should be float. I'll fix that and also add some more tests to up the coverage for spec.py